controllers/main.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
+
+import time
+
+'''
+In this game we have two role - invader and hero. Both can bullet.
+
+Invaders are located at the top of game field and always move from the right side to the left and revert.
+Also they slowly move to the bottom.
+
+Main hero is located at the centre at the bottom. He can move to the left and to the right.
+If he bullet some invader, his bonus is grow.
+When invader bullet to main hero, the main hero's life is decrease.
+'''
+
+
+class Unit(object):
+
+    def __init__(self, unit_filename, bullet_filename, x, y, bonus=0, speed=5):
+        self.image = unit_filename
```

Comment:
`self.image` is intentionally an object on image. If you need a property for image file name please name it `self.image_filename`